src/blockstream_service.rs

```diff
@@ -108,76 +106,62 @@ impl Service for BlockstreamService {
 #[cfg(test)]
 mod test {
     use super::*;
-    use crate::entry::{Entry, EntryMeta};
+    use crate::blocktree::{create_new_ledger, get_tmp_ledger_path};
+    use crate::entry::{create_ticks, Entry};
     use chrono::{DateTime, FixedOffset};
     use serde_json::Value;
+    use solana_sdk::genesis_block::GenesisBlock;
     use solana_sdk::hash::Hash;
     use solana_sdk::signature::{Keypair, KeypairUtil};
     use solana_sdk::system_transaction::SystemTransaction;
+    use std::sync::mpsc::channel;
 
     #[test]
     fn test_blockstream_service_process_entries() {
         let ticks_per_slot = 5;
         let leader_id = Keypair::new().pubkey();
 
+        // Set up genesis block and blocktree
+        let (mut genesis_block, _mint_keypair) = GenesisBlock::new(1000);
+        genesis_block.ticks_per_slot = ticks_per_slot;
+        let (ledger_path, _last_id) = {
```

Comment:
@mvines , I attempted to use `create_new_tmp_ledger` (macro) instead, but cargo kept reporting it as not in scope, even though my `use` line above was identical to other modules. Do you know why this would be?

Reply:
#3007